lms/templates/api_admin/api_access_request_email_denied.txt

```diff
@@ -0,0 +1,7 @@
+## mako
+Dear ${name},
+
+Your request for access to the edX Course Catalog API has been denied. If you think this is an error, or for other questions about using this API, contact ${support_email_address}.
+
+Thanks!
```

Comment:
This salutation is a bit too cheery for a rejection notice. Might be best to leave it out.